admin_guide/diagnostics_tool.adoc

```diff
@@ -157,36 +180,173 @@ A failure from this check indicates that a significant amount of space in etcd
 is being taken up by {product-title} image data, which can eventually result in
 your etcd cluster crashing.
 
-A user-defined limit may be set by passing the variable
-`etcd_max_image_data_size_bytes=400000000` to the `openshift_health_checker`
-role.
+A user-defined limit may be set by passing the `etcd_max_image_data_size_bytes`
+variable. For example, setting `etcd_max_image_data_size_bytes=40000000000` will
+cause the check to fail if the total size of image data stored in etcd
+exceeds 40 GB.
+
+|`etcd_traffic`
+|This check detects higher than normal traffic on an etcd host. It fails if a
+`journalctl` log entry with an etcd sync duration warning is found.
+
+For further information on improving etcd performance, see
+xref:../scaling_performance/host_practices.adoc#scaling-performance-capacity-host-practices-etcd[Recommended Practices for {product-title} etcd Hosts] and the
+link:https://access.redhat.com/solutions/2916381[Red Hat Knowledgebase].
 
 |`etcd_volume`
 |This check ensures that the volume usage for an etcd cluster is below a maximum
 user-specified threshold. If no maximum threshold value is specified, it is
 defaulted to `90%` of the total volume size.
 
-A user-defined limit may be set by passing the variable
-`etcd_device_usage_threshold_percent=90` to the `openshift_health_checker` role.
+A user-defined limit may be set by passing the
+`etcd_device_usage_threshold_percent` variable.
 
 |`docker_storage`
-|Only runs on hosts that depend on the *docker* damon (nodes and containerized
+|Only runs on hosts that depend on the *docker* daemon (nodes and containerized
 installations). Checks that *docker*'s total usage does not exceed a
 user-defined limit. If no user-defined limit is set, *docker*'s maximum usage
-threshold defaults to 90% of the total size available. The threshold
-limit for total percent usage can be set with a variable in your inventory file:
-`max_thinpool_data_usage_percent=90`.
-|===
+threshold defaults to 90% of the total size available.
```

Comment:
It also checks that docker storage is configured in a supportable way. Meaning, with a storage driver and backing storage device that are supported for usage with Docker.

In particular, the default configuration of Docker storage is not supported. I'm not sure how much to say about this; probably worth linking to any docs we have on Docker storage configuration.